p2p/src/authenticated/config.rs

```diff
@@ -63,11 +63,15 @@ pub struct Config<C: Scheme> {
     /// Quota for incoming connections across all peers.
     pub allowed_incoming_connection_rate: Quota,
 
-    /// Frequency to attempt to dial known addresses.
+    /// Frequency at which we make a new dial attempt to a peer.
     pub dial_frequency: Duration,
 
-    /// Quota for peers to dial.
-    pub dial_rate: Quota,
+    /// Frequency at which we will fetch a new list of dialable peers.
+    pub query_frequency: Duration,
```

Comment:
nit: would add a note here that this is the fastest we will ever try to redial a given peer.